src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -479,7 +479,7 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
             individual.npe01__SYSTEMISINDIVIDUAL__c = true;
             //individual.Type = CAO_Constants.BUCKET_ORGANIZATION_TYPE;
             individual.npe01__SYSTEM_AccountType__c = CAO_Constants.BUCKET_ORGANIZATION_TYPE;
-            insert individual;
+            UTIL_DMLService.insertRecord(individual, true);
```

Comment:
seems like UTIL_DMLService.insertRecord(individual) would have been the logical replacement for straight insert calls, but as I call out in DMLService, I think it should default to allOrNone to be consistent with straight insert calls.